### PR TITLE
Update core-foundation-sys to 0.3

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15365)

### CODE REVIEW

File: components/servo/lib.rs [hunk 1]
```diff
@@ -184,7 +184,7 @@ impl<Window> Browser<Window> where Window: WindowMethods + 'static {
                 clear_framebuffer: true,
                 clear_color: webrender_traits::ColorF::new(1.0, 1.0, 1.0, 1.0),
                 render_target_debug: false,
-            })
```
Comment (mbrubeck) [inline components/servo/lib.rs]: Can we change this to `.expect(...)`, please?

File: Cargo.lock [hunk 2]
```diff
@@ -1440,16 +1417,11 @@ dependencies = [
  "profile_traits 0.0.1",
  "script_traits 0.0.1",
  "servo_url 0.0.1",
- "webrender_traits 0.11.0 (git+https://github.com/servo/webrender)",
+ "webrender_traits 0.13.0 (git+https://github.com/servo/webrender)",
 ]
 
 [[package]]
 name = "lazy_static"
-version = "0.1.16"
```
Comment (mbrubeck) [inline Cargo.lock]: You can remove `lazy_static` from the ignore list in `servo-tidy.toml`.

**Comment by glennw** [comment]
@mbrubeck @nox This PR (not merged yet) https://github.com/servo/webrender/pull/831 fixes the  tex_image_2d_mipmap.html test failure. We still don't have the performance regression fix I mentioned yet, but we may just do the update when the fix above lands anyway.

**Pull request closed without merging**